zmq/src/zmq.ml

```diff
@@ -232,8 +232,11 @@ module Socket = struct
     validate_string_length 1 255 identity "set_identity";
     set_bytes_option socket ZMQ_IDENTITY identity
 
+  let maximal_buffer_length = 256
```

Comment:
If we keep the buffer size in the c-stub at 256 bytes and want to rtie `\0` at the end of the buffer, this value should be lowered by one.